src/utils/find-deepest-node.js

```diff
@@ -7,7 +7,7 @@
  */
 
 function findDeepestNode(element) {
-  return element.firstChild
+  return element && element.firstChild
```

Comment:
@doodlewind do you know where this was causing an issue?

Reply:
This should have been done with another PR. When element itself is null, IMO returning null is somehow better then throwing an error. This happens when removing nodes in a loose schema. Sorry for not providing reproduction here, while if this is not suitable to be merged here, it's okay to close this with another pull request.😅